backend/src/main/java/com/bakdata/conquery/models/query/queryplan/specific/ValidityDateNode.java

```diff
@@ -49,6 +49,6 @@ public void nextTable(QueryExecutionContext ctx, Table currentTable) {
 	
 	@Override
 	public String toString() {
-		return "ValidityDateNode [validityDateColumn=" + validityDateColumn + ", getChild()=" + getChild() + "]";
```

Comment:
Mal machst du .getClass().getSimpleName mal Lombok, mal so

Reply:
Das Problem ist, dass Lombok nicht mit Vererbung umgehen kann in keine von beiden Richtungen. Ich habe Lombok gemacht wo es Sinn macht, und da von Hand gerollt, wo es muss. (also da wo Felder von Child und Parent reinkommen sollten). Wenn ich eine Klasse hatte, die abstract ist und gut funktioniert, habe ich getClass.getSimpleName gemacht, weil es dann auf alle Kinder geht.